amp/WEB-INF/src/org/digijava/module/aim/audit/AuditActivityInfo.java

```diff
@@ -0,0 +1,34 @@
+package org.digijava.module.aim.audit;
+
+import org.digijava.module.aim.dbentity.AmpTeamMember;
+
+public class AuditActivityInfo {
```

Comment:
Please add here method that encapsulates try/finally logic to avoid code repetition and misuse of this class.
Also remove getters/setter to field & static variable and expose only `runInContext()` and `getTeamMember()` methods.